### PR TITLE
add AKS container images to VHD build script

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
add AKS container images to VHD build script
```

### CODE REVIEW

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     pullContainerImage "docker" "nvidia/k8s-device-plugin:${NVIDIA_DEVICE_PLUGIN_VERSION}"
 done
 
+TUNNELFRONT_VERSIONS="v1.9.2-v4.0.4"
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: // TODO: cleanup images not needed at runtime in https://github.com/Azure/acs-engine/blob/cleanup-images/parts/k8s/kubernetesinstalls.sh#L220

File: packer/install-dependencies.sh [hunk 2]
```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     pullContainerImage "docker" "nvidia/k8s-device-plugin:${NVIDIA_DEVICE_PLUGIN_VERSION}"
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: is this missing a "v" for version? (I know it's not from this PR but just noticed)

File: packer/install-dependencies.sh [hunk 3]
```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
...
 done
 
+TUNNELFRONT_VERSIONS="v1.9.2-v4.0.4"
+for TUNNELFRONT_VERSION in ${TUNNELFRONT_VERSIONS}; do
+    pullContainerImage "docker" "docker.io/deis/hcp-tunnel-front:${TUNNELFRONT_VERSION}"
+done
+
+KUBE_SVC_REDIRECT_VERSIONS="v1.0.2"
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: let's make the version var "1.0.2" and put the "v" in the image string below to be consistent with the rest of the file and avoid human error

File: packer/install-dependencies.sh [hunk 4]
```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
...
+for KUBE_SVC_REDIRECT_VERSION in ${KUBE_SVC_REDIRECT_VERSIONS}; do
+    pullContainerImage "docker" "docker.io/deis/kube-svc-redirect:${KUBE_SVC_REDIRECT_VERSION}"
+done
+
+NGINX_VERSIONS="1.13.12-alpine"
+for NGINX_VERSION in ${NGINX_VERSIONS}; do
+    pullContainerImage "docker" "nginx:${NGINX_VERSION}"
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: this one doesn't have the "v" for version?

Comment (jackfrancis) [reply packer/install-dependencies.sh]: correct

File: packer/install-dependencies.sh (hunk 1 above)
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: what does the dash mean in "v1.9.2-v4.0.4"? is it one version? or should it be a space?

File: packer/install-dependencies.sh (hunk 2 above)
Comment (jackfrancis) [reply packer/install-dependencies.sh]: there is no "v" prefix for this image

File: packer/install-dependencies.sh (hunk 1 above)
Comment (jackfrancis) [reply packer/install-dependencies.sh]: Talk to the publisher. :) It's just the way it's versioned.

Comment (jackfrancis) [reply packer/install-dependencies.sh]:
```
Containers:
  tunnel-front:
    Container ID:   docker://bbf2a60b072c724b88132537a0caf961c467c73790c8e44f12a9467ba402e85e
    Image:          docker.io/deis/hcp-tunnel-front:v1.9.2-v4.0.4
```

Comment (jackfrancis) [reply packer/install-dependencies.sh]: Actually, given that this is already oddly versioned as you noticed, let's skip the whole "v" removal thing and just keep this as an oddball.

Comment (CecileRobertMichon) [reply packer/install-dependencies.sh]: sounds good

**Comment by CecileRobertMichon** [comment]
https://msazure.visualstudio.com/One/_build/results?buildId=18402607&_a=summary&view=logs

**Pull request merged**